CharacterCreation/Class-Specific Documentation/Hunters_Dog.txt

```diff
@@ -0,0 +1,64 @@
+==============================
+Hunter's Dog Rules and Stats
+==============================
+
+
+===== Dog Rules =====
+
+    The Dog cannot use weapons, armor nor non-food items. 
+
+    The Dog is smaller than most combatants and moves very quickly. If the Dog 
+has moved at least 8m on their last turn, they are considered to be in partial 
+cover from all angles. 
+
+    The Dog has no guard in melee. Instead, up to four times per round the dog 
+may roll a reflex save of DC 70 to dodge a melee attack.
+
+===== Dog Stats =====
+
+Health:     80
+Strength:   4
+Perception: 12
+Fortitude:  3
+Charisma:   8
+Intelligence: 2
+Dexterity:  12
+Luck:       5
+Will:       10
+Reflex:     36
+Shock:      -2
+Move Speed Per Action: 6
+Death Saving Throws: 3
+Nanites:    0
+
+    The Dog has a +(level x 4) bonus to Investigation Checks and Awareness Saves 
+that their sense of smell can help with. 
+
+    The Dog can use tricks like "lay down," "shake," "fetch," or "chase your 
+tail" to charm or distract characters in the area. When they do, the target must
+succeed on a Will save opposed by the dog's Performance (Cha) roll.
+
+    The Dog's primary weapon is their bite. Their melee roll is 
+6 + (level/4) + roll. On a successful hit, the target is grappled with the Dog
+in control, and the target must make a Shock Save of DC 76. If the target 
+succeeds on the shock save, they stay standing with the Dog in control of the 
+grapple. If they fail the Shock Save, the target is knocked prone also with the
+dog in control of the grapple. 
+	
+	Every time the Dog makes a melee attack while it is in control of a
+grapple, it deals 60 damage. Dog attacks usually involve the dog either making
+subsequent bites, and/or shaking their head viciously to shred what they're 
+currently biting on to ribbons.
```

Comment:
Grammar suggestion: Dogs have two primary methods of attacking: biting a new section of the target, or shaking their head viciously to tear apart a portion they are currently biting.